### PR TITLE
nitrile gloves buff

### PR DESCRIPTION
**Pull request by manofpepsi**
## About The Pull Request

makes the nitrile gloves half the use time of the syringes, patches, pills and medigels when you use them on someone.

## Why It's Good For The Game

makes the nitrile gloves worthwhile wearing as paramedic or cmo/md.

## Changelog
:cl: Pepsiman0
balance: nitrile gloves halve the use time of the syringes, pills, patches and medigels
/:cl:

### CODE REVIEW

**Comment by D4C-420** [comment]
finally a good glove PR.

**Comment by Fikou** [comment]
bad changelog

**Comment by Time-Green** [comment]
Still a really bad changelog

**Comment by Time-Green** [comment]
Still a really bad changelog, you can at least try

**Comment by Time-Green** [comment]
Now that's a good changelog! 👍

File: code/__DEFINES/nitrile.dm [hunk 1]
```diff
@@ -0,0 +1,5 @@
+///normal use time for syringes, pills, patches and medigels
+#define NORMAL_USE 3 SECONDS
```
Comment (Mothblocks) [inline code/__DEFINES/nitrile.dm:2]: These are bad names (should be scoped to nitrile gloves, i.e. `NORMAL_USE_NO_NITRILES`), and it's bad that this is inside a file called "nitrile.dm", when it is for syringes, pills, patches, and medigels.

I think it would be better if instead of this, you just had a define for a multiplier to use--`#define NITRILE_GLOVE_SPEED_MULTIPLIER 0.5`

File: tgstation.dme [hunk 2]
```diff
@@ -23,6 +23,7 @@
 #include "code\__DEFINES\_protect.dm"
 #include "code\__DEFINES\_tick.dm"
 #include "code\__DEFINES\access.dm"
+#include "code\__DEFINES\nitrile.dm"
```
Comment (Mothblocks) [inline tgstation.dme:26]: Don't edit this file manually, otherwise every PR from this point forward is going to have this auto-fixed.

**Review by Mothblocks — CHANGES_REQUESTED** [review]

File: tgstation.dme (hunk 2 above)
Comment (optimumtact) [reply tgstation.dme:26]: you can edit it manually you just need to sort your includes

**Comment by moocowswag** [comment]
Wont this mean you can shove stun someone and then bluespace syringe them extremely consistently? (You already can but it requires aggro grabbing while injecting and then they have to fail the chance to break)

**Comment by Mothblocks** [comment]
while thats a good point, if youre close range you can already use a syringe gun, so it doesnt affect balance too greatly i think?

**Comment by moocowswag** [comment]
I think I would even use nitrile gloves over a syringe gun when I can.

Mainly due to the fact that the syringe gun tends to destroy its syringes and I`d like to be able to reuse my funny bluespace syringe which is made of materials that miners are often too lazy to get an abundant amount of.

**Comment by ATH1909** [comment]
*evil chuckling*

File: code/modules/clothing/gloves/color.dm [hunk 3]
```diff
@@ -204,7 +204,7 @@
 	desc = "Pricy sterile gloves that are thicker than latex. Excellent grip ensures very fast carrying of patients."
 	icon_state = "nitrile"
 	inhand_icon_state = "nitrilegloves"
-	clothing_traits = list(TRAIT_QUICKER_CARRY)
+	clothing_traits = list(TRAIT_QUICKER_CARRY, TRAIT_FASTMED)
 	transfer_prints = FALSE
 
 /obj/item/clothing/gloves/color/latex/nitrile/infiltrator
```
Comment (ATH1909) [inline code/modules/clothing/gloves/color.dm:210]: you should probably mention in the changelog that infiltrator gloves inherit this

**Comment by GuillaumePrata** [comment]
If stun + syringe is a balance issue, wouldn't a check for being jittery and others help with that at least?
I know we already have some of those around for what I assume is the same goal.

File: code/modules/clothing/gloves/color.dm (hunk 3 above)
Comment (manofpepsi) [reply code/modules/clothing/gloves/color.dm]: infiltrator gloves are now /obj/item/clothing/gloves/color/infiltrator

File: code/modules/clothing/gloves/color.dm [hunk 4]
```diff
@@ -201,20 +201,21 @@
...
+	clothing_traits = list(TRAIT_QUICKER_CARRY, TRAIT_FASTMED)
 	transfer_prints = FALSE
 
-/obj/item/clothing/gloves/color/latex/nitrile/infiltrator
+/obj/item/clothing/gloves/color/infiltrator
 	name = "infiltrator gloves"
 	desc = "Specialized combat gloves for carrying people around. Transfers tactical kidnapping knowledge into the user via nanochips."
 	icon_state = "infiltrator"
 	inhand_icon_state = "infiltrator"
-	siemens_coefficient = 0
```
Comment (ATH1909) [inline code/modules/clothing/gloves/color.dm:215]: why did you remove the shock immunity from infiltrator gloves???

Comment (manofpepsi) [reply code/modules/clothing/gloves/color.dm:215]: whoops, good catch.

**Comment by Farquaar** [comment]
This makes a lot of sense. Good buff.
